e2e/tests/nightly/admin/user-admin-table.spec.ts

```diff
@@ -0,0 +1,45 @@
+import UserAdminTablePage from 'app/page/admin/user-admin-table-page';
+import { signInWithAccessToken } from 'utils/test-utils';
+import { config } from 'resources/workbench-config';
+import navigation, { NavLink } from 'app/component/navigation';
+import AdminTable from 'app/component/admin-table';
+
+describe('User Admin Table', () => {
+  const userEmail = 'admin_test';
+
+  beforeEach(async () => {
+    await signInWithAccessToken(page, config.ADMIN_TEST_USER);
+    await navigation.navMenu(page, NavLink.USER_ADMIN);
+  });
+
+  test('verify user-admin-table can access the bypass toggles and navigate to user-admin-profile', async () => {
```

Comment:
this is the first test in the old file user-admin-ui.spec with a new description